azurerm/resource_arm_securitycenter_subscription_pricing.go

```diff
@@ -0,0 +1,94 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/security/mgmt/2017-08-01-preview/security"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/suppress"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmSecurityCenterSubscriptionPricing() *schema.Resource {
```

Comment:
thinking about this longer term - I think this'd make more sense as a nested element on the `azurerm_subscription` resource (but this is fine as an independent resource for the moment) 🤔